protostar/testing/test_collector.py

```diff
@@ -77,7 +77,9 @@ def fnmatch_any(name: str, pats: Iterable[str]) -> bool:
 
 
 class FunctionNameGetter(Protocol):
-    def __call__(self, file_path: Path) -> List[str]:
+    def __call__(
+        self, file_path: Path
+    ) -> Union[List[str], List[Tuple[str, AvailableGas]]]:
```

Comment:
```suggestion
    ) -> Union[list[str], list[tuple[str, AvailableGas]]]:
```

Reply:
Done

Reply:
is it, can you recheck? I think it's still the way it was.

Reply:
oops, changed it in a different place, now it's ready

Reply:
we should consider moving from this typing in all modules, perhaps in a single issue